contents/docs/integrations/js-integration.mdx

```diff
@@ -156,9 +163,26 @@ posthog.init('[your api key]', {
 });
 ```
 
+#### Multiple IDs
+
+If you use multiple distinct IDs for the same user (e.g. a logical ID and a UUID), you may want to set both for the same user. This way, if PostHog receives events for either ID, it will properly match them to the same person. This is also helpful if you want to associate anonymous IDs with an identified person (we actually do this automatically when you call `.identify`). To associate multiple IDs to the same person, you do an **alias** call, as shown below.
+
+```js
+posthog.alias('[new ID]', '[original ID]')
+```
+
+The original ID will be kept as the default, but any events sent to either will be received under the same person.
+
 ### Reset After Logout
 
-If a user is logged out, you probably want to call `reset()` to unset any `distinct_ids`. This is especially important if your users are sharing a computer, as otherwise all of those users will be grouped together into 1 user due to shared cookies between sessions.
+If a user is logged out, you probably want to call `reset()` to unset any `distinct_ids`.
```

Comment:
I know you're being consistent with the past copy here but we should probably keep this consistent throughout, i.e. if we use `identify`, let's also use `reset`

Reply:
also maybe `posthog.reset` in this case